### PR TITLE
Add command documentation

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This is my proposal for the command documentation.

Of course, not all commands are added yet, as I want you to ACK the
design first, SYN?

Targets #445.

### CODE REVIEW

**Comment by neithernut** [comment]
To tell the user that arguments can also be stack positions is unneccessary for regular commands. Maybe you should have subsections for regular commands (subsectioning is also missing btw.) and _one_ section for special commands.

**Comment by matthiasbeyer** [comment]
Good points.

**Comment by manuelmessner** [comment]
ACK

**Comment by neithernut** [comment]
Split up regular commands. We _do_ have a structure here, e.g. arithmetical, logical, bitwise etc.

**Comment by neithernut** [comment]
Nah, you can ommit the "regualr commands" section completely in favor for subsections for the subsubsections you introduces. This level is unneccessary and may even confuse the user who doesn't have any knowledge about the actual implementation.

**Comment by neithernut** [comment]
Now this looks acceptable. You can now proceed adding actual documentation.

**Comment by neithernut** [comment]
Er... and _please_ add a blank line at EOF!

File: doc/user/src/api/commands.tex [hunk 1]
```diff
@@ -1,3 +1,155 @@
...
+                {
+                    Checks whether the argument is false, whereas only a NIL and
+                    a Boolean which is false evaluate to false.
+                }
+
+        \command{land}
+                {N : Value}
+                {Boolean}
+                {
+                    Logical and between all arguments, whereas only NIL and a
```
Comment (xVin) [inline doc/user/src/api/commands.tex]: maybe AND in uppercase?
same could be done for all the other logical commands

Comment (matthiasbeyer) [inline doc/user/src/api/commands.tex]: Good point. @neithernut What format would you suggest? Uppercase for the command names, or in quotes? Maybe something else?

Comment (neithernut) [inline doc/user/src/api/commands.tex]: It just took me, like, a minute, to get what this sentence was about. The `and` should _definitely_ be uppercase or even be formatted as an inline listing.

**Comment by matthiasbeyer** [comment]
Rewrote to use uppercase names. We might add a `$...$` environment (math environment) around them to lift them a bit from the text around them... I'm not sure about this.

**Comment by neithernut** [comment]
Using a math environment here is _not_ acceptable.
Maybe we will change this in the future, but it's acceptable now as it is.
ACK.

**Comment by matthiasbeyer** [comment]
Please re-read your last sentence... 

And please review carefully if the wording is appropriate.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**